packages/webview_flutter/webview_flutter/lib/src/webview.dart

```diff
@@ -83,6 +83,7 @@ class WebView extends StatefulWidget {
     this.javascriptChannels,
     this.navigationDelegate,
     this.gestureRecognizers,
+    this.bouncesEnabled = true,
```

Comment:
Android has overscroll as well, and API to control it for web views, so this should be given a more generic name.

Reply:
It would be nice if you suggest the name instead of just simple reject it.

Reply:
I didn't reject anything, I just left a review comment.

I would suggest something like `enableOverscroll`, since overscroll is the term Flutter uses for this.